src/panels/lovelace/editor/view-editor/hui-dialog-edit-view.ts

```diff
@@ -212,20 +217,24 @@ export class HuiDialogEditView extends LitElement {
         <mwc-button @click=${this.closeDialog} slot="primaryAction"
           >${this.hass!.localize("ui.common.cancel")}</mwc-button
         >
-        <mwc-button
-          slot="primaryAction"
-          ?disabled=${!this._config || this._saving}
-          @click=${this._save}
-        >
-          ${this._saving
-            ? html`<ha-circular-progress
-                active
-                size="small"
-                title="Saving"
-              ></ha-circular-progress>`
-            : ""}
-          ${this.hass!.localize("ui.common.save")}</mwc-button
-        >
+        ${this._dirty
```

Comment:
Let's disable the button instead of not rendering it

Reply:
Currently in the Entity editor dialog, the 'Close' button changes to 'Save' if there is a change. The View editor (tabs) dialog the button is always `Save`.

In my UX Hitlist I made a comment I don't like the buttons `Cancel` and `Close` next to each other. They're the same. It's also a best practice to alway keep the submit button enabled, because people want to make sure things are saved and we can show UI errors. 

I would likte to suggest to have a `Cancel` and `Save` button that are always enabled. Like we do on the View editor dialog.